cli/src/config/templates.toml

```diff
@@ -41,10 +41,24 @@ tag_list = '''
 label("tag", name) ++ format_ref_targets(self) ++ "\n"
 '''
 
+# TODO: "op_log" is not a great label name for op_summary. We could replace it
+# with `operation` and insert this new label throughout the op log templates.
 op_summary = '''
-if(root,
-  separate(" ", self.id().short(), label("root", "root()")),
-  separate(" ", self.id().short(), format_time_range(self.time()), self.description().first_line())
+label(
+  "op_log",
+  concat(
+    self.id().short(),
+    if(!root, concat(
+       ", ",
+       label("time", "finished "),
+       format_timestamp(self.time().end())
+    )),
```

Comment:
nit: I would omit "finished " because it's shorter and I don't care whether the timestamp is start or end. As I don't care, it might be better to use the start time. I think the start time is displayed in `op log` if you disabled relative timestamps.

Reply:
>  I don't care whether the timestamp is start or end. As I don't care, it might be better to use the start time. I think the start time is displayed in op log if you disabled relative timestamps.

Is there a situation you are thinking of where the start is more useful than the end?

My current feeling is that I do care somewhat, and I think that when it matters, the end is more important. It's the moment where the repository was actually modified with the result of the operation. For example, as I mentioned above, if I was writing a description in `jj describe` for an hour, saved it, and then undid it, it'd be easier for me to remember the end moment. Another way to see that it's important is that aborted operations have a start time but no end time, and lacking the end time is related to the fact that they are not important.

That could be considered a separate issue from this PR, though. I thought we display both the start and end time in the op log, but it might make sense to display either the time ago + duration or the ending time + duration.

>  I would omit "finished " because it's shorter

I could say "ended" or "from" if you prefer shorter. This brings us back to the recurring argument where I like using English words and you do not like it.

Not using English would make more sense to me if we didn't have "Undid operation" before the summary, but something like "Operation undone:". This would require making changes to several commands, which I'd prefer to consider outside of the scope of this PR.

---------------------

If none of these work for you, I'll just adjust this to your liking tomorrow (I'm not finishing it today in any case). We can change it later, ultimately I just want something that looks OK in the demo.

Reply:
> Is there a situation you are thinking of where the start is more useful than the end?

No. I'm just saying that using start time is more consistent because it's displayed in "op log". Maybe we can change "op log" to use the end time?

> > I would omit "finished " because it's shorter
> 
> I could say "ended" or "from" if you prefer shorter. This brings us back to the recurring argument where I like using English words and you do not like it.
> 
> Not using English would make more sense to me if we didn't have "Undid operation" before the summary, but something like "Operation undone:". This would require making changes to several commands, which I'd prefer to consider outside of the scope of this PR.

I prefer not using English word there because the summary template can be embedded in various places. I think the intent of "Undid operation" message is `Undid operation <quote>the-operation</quote>`.

Reply:
> Maybe we can change "op log" to use the end time?

Yeah, I think that might be good. Might be a separate PR.

> I think the intent of "Undid operation" message is Undid operation <quote>the-operation</quote>.

I'll experiment with `Undid operation: op_summary_template` tomorrow.

Reply:
"tomorrow" turned out to be optimistic, but here it is.

I changed the times to the start time for now, but I intend to change it in a separate PR.